roles/openshift_clock/tasks/main.yaml

```diff
@@ -3,7 +3,7 @@
   openshift_facts:
     role: clock
     local_facts:
-      enabled: "{{ openshift_clock_enabled | default(None) }}"
+      enabled: "{{ openshift_clock_enabled | default(True) }}"
```

Comment:
I'm good with removing the dependency on openshift_facts, just need to make sure that we replace the existing functionality.